tests/QrPaymentTest.php

```diff
@@ -6,16 +6,16 @@
 use DateTimeInterface;
 use Endroid\QrCode\QrCode;
 use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
 use Rikudou\Iban\Iban\IBAN;
 use rikudou\SkQrPayment\Exception\InvalidTypeException;
 use rikudou\SkQrPayment\Exception\QrPaymentException;
 use rikudou\SkQrPayment\Payment\QrPaymentOptions;
 use rikudou\SkQrPayment\QrPayment;
-use PHPUnit\Framework\TestCase;
-use rikudou\SkQrPayment\Xz\XzBinaryLocator;
+use rikudou\SkQrPayment\Xz\LinuxXzBinaryLocator;
 use TypeError;
 
-class QrPaymentTest extends TestCase
+final class QrPaymentTest extends TestCase
```

Comment:
All the changes (except line 209) are out of scope of this PR, those are all code style changes which should be done in separate PR.